src/server/db_slice.cc

```diff
@@ -224,8 +224,21 @@ unsigned PrimeEvictionPolicy::Evict(const PrimeTable::HotspotBuckets& eb, PrimeT
     }
 
     DbTable* table = db_slice_->GetDBTable(cntx_.db_index);
+    auto& lt = table->trans_locks;
+    string tmp;
+    string_view key = last_slot_it->first.GetSlice(&tmp);
+    if (lt.find(KeyLockArgs::GetLockKey(key)) != lt.end())
+      return 0;
+
     PerformDeletion(last_slot_it, db_slice_->shard_owner(), table);
     ++evicted_;
+
+    // delete the key in replicas.
```

Comment:
Journal can be used for several things, one of them is replication, another thing can be log journal changes to file, i.e AOF redis feature.
Please just fix the comment to say log to journal

Reply:
fixed